### PR TITLE
Fix image fcm

### PR DESCRIPTION
**Pull request by SebastienMelki**
@appleboy  This ties in to PR https://github.com/appleboy/go-fcm/pull/22 from gofcm

### CODE REVIEW

**Comment by appleboy** [comment]
@SebastienMelki Please fix the conflicts.

**Comment by SebastienMelki** [comment]
> Please also update the go-fcm package.

I fixed the conflicts, but how can I update the go-fcm package, you need to make a new release there no? It's still on v0.1.4

**Comment by appleboy** [comment]
@SebastienMelki I created new tag v0.1.5

see https://github.com/appleboy/go-fcm/releases/tag/v0.1.5

**Comment by appleboy** [comment]
@SebastienMelki any updates?

**Comment by SebastienMelki** [comment]
> @SebastienMelki any updates?

updated go-fcm, sorry for the delay

File: go.mod [hunk 1]
```diff
@@ -55,4 +55,5 @@ require (
 	gopkg.in/go-playground/validator.v9 v9.30.2 // indirect
 	gopkg.in/redis.v5 v5.2.9
 	gopkg.in/yaml.v2 v2.2.7 // indirect
+	honnef.co/go/tools v0.0.0-20190523083050-ea95bdfd59fc // indirect
```
Comment (appleboy) [inline go.mod]: Why need this?

Comment (appleboy) [reply go.mod]: fixed in https://github.com/appleboy/gorush/commit/cee47bdc2c56d875c97e9d5dbdd76555996a513b

**Pull request merged**